content/ui-testing-handbook/react/en/accessibility-testing.md

```diff
@@ -0,0 +1,288 @@
+
+## Why automation?
+
+Before we begin, let’s examine common types of disabilities: visual, hearing, mobility, cognition, speech, and neurological. These user disabilities yield app requirements like:
+
+- ⌨ Keyboard navigation
+- 🗣 Screen reader support
+- 👆 Touch-friendly
+- 🎨 High enough colour contrast
+- ⚡️ Reduced motion
+- 🔍 Zoom
+
+In the past, you’d verify each of these requirements by checking every component across a combination of browsers, devices, and screen readers. But that’s impractical to do by hand because apps have dozens of components and are constantly updating the UI.
+
+## Automation speeds up your workflow
+
+Automated tools audit the rendered DOM against a set of heuristics based on [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) rules and other industry-accepted best practices. They act as the first line of QA to catch blatant accessibility violations.
+
+For example, Axe, on average, finds [57% of WCAG issues automatically](https://www.deque.com/blog/automated-testing-study-identifies-57-percent-of-digital-accessibility-issues/). That allows teams to focus their expert resources on the more complex issues that require manual review.
+
+Many teams use the [Axe library](https://github.com/dequelabs/axe-core) because it integrates with most existing test environments. For example, the [Twilio Paste](https://github.com/twilio-labs/paste) team uses the [jest-axe integration](https://github.com/twilio-labs/paste/blob/cd0ddad508e41cb9982a693a5160f1b7866f4e2a/packages/paste-core/components/checkbox/__tests__/checkboxdisclaimer.test.tsx#L40). Whereas the Shopify Polaris & Adobe Spectrum teams use the [Storybook addon](https://storybook.js.org/addons/@storybook/addon-a11y) version.
+
+## Accessibility testing workflow
+
+By running these checks throughout the development process, you shorten the feedback loop and fix issues faster. Here’s what the workflow looks like:
+
+1.  👨🏽‍💻 **During development:** use Storybook to focus on one component at a time. Use the A11y addon to simulate vision defects and run an accessibility audit at the component level.
+2.  ✅ **For QA:** integrate the Axe audit into your functional testing pipeline. Run checks on all components to catch regressions.
+
+![](/ui-testing-handbook/a11y-workflow.png)
+
+Let’s see this workflow in action.
+
+### Install the accessibility addon
+
+Storybook’s Accessibility runs Axe on the active story. It visualizes the test results in a panel and outlines all DOM nodes that have a violation.
+
+![](/ui-testing-handbook/a11y-testing.gif)
+
+To install the addon, run: `yarn add -D @storybook/addon-a11y`. Then, add `'@storybook/addon-a11y'` to the addons array in your `.storybook/main.js`:
+
+```javascript:title=.storybook/main.js
+const path = require('path');
+
+const toPath = (_path) => path.join(process.cwd(), _path);
```

Comment:
I don't think these lines or the `webpackFinal` line add anything to this tutorial. Suggest removing them.